modules/kv/1.0/azureDeploy.json

```diff
@@ -66,18 +66,16 @@
                 "description": "Specifies the SKU for the vault"
             }
         },
-        "oms-workspace-resourceGroup": {
+        "la-resourceId": {
             "type": "string",
-            "minLength": 1,
             "metadata": {
-                "description": "Name of the Resource Group housing the OMS workspace."
+                "description": "Log Analytics workspace resource identifier"
             }
         },
-        "oms-workspace-name": {
+        "storageAccount-resourceId": {
```

Comment:
storageAccount-resourceId and la-resourceId pointing to the same resource id based on description. Can we combine it to one?

Reply:
Based on module dependencies, these two are different resources. I'll update the description of storageAccount-resourceId